polyA/_runners.py

```diff
@@ -238,7 +251,7 @@ def run_full(
         active_cells,
         consensus_matrix,
     ) = fill_consensus_position_matrix(
```

Comment:
Right now this returns non_empty_cols but we should just make non_empty_columns = [col for col in range(column_count)] instead (for now so things don't break)